packages/main/src/Calendar.ts

```diff
@@ -182,8 +182,10 @@ type SpecialCalendarDateT = {
  * @csspart day-cell-selected-between - Used to style the day cells in between of selected dates in range.
  * @csspart month-cell - Used to style the month cells.
  * @csspart month-cell-selected - Used to style the month cells when selected.
+ * @csspart month-cell-selected-between - Used to style the day cells in between of selected months in range.
  * @csspart year-cell - Used to style the year cells.
  * @csspart year-cell-selected - Used to style the year cells when selected.
+ * @csspart year-cell-selected-between - Used to style the day cells in between of selected years in range.
```

Comment:
export the parts in the hbs, because currently we are unable to use them

Reply:
The parts are only an extension to those added with change https://github.com/SAP/ui5-webcomponents/pull/9383, which is a feature only for ui5-calendar. If there's a specific request to add them to ui5-datepicker and other picker controls, we can consider adding them then.